src/Angular.js

```diff
@@ -354,6 +354,8 @@ function baseExtend(dst, objs, deep) {
       if (deep && isObject(src)) {
         if (isDate(src)) {
           dst[key] = new Date(src.valueOf());
+        } else if (isElement(src)) {
+          dst[key] = src[0] ? jqLite(src).clone()[0] : jqLite(src).clone();
```

Comment:
A little late to the party, but shouldn't it be the other way around?
Namely: `!src[0] ? jqLite(src).clone()[0] : jqLite(src).clone();`

(In fact, I would rather expect: `!src[0] ? jqLite(src).clone()[0] : src.clone();`)